docs/roadmap.md

```diff
@@ -1,38 +1,120 @@
+<!-- markdownlint-disable MD024 -->
+
 # Roadmap
 
-The aim is to deliver a Minimum Viable Product (MVP). In a nutshell, an MVP
-seeks to collect the maximum amount of validated learning about our users in a
-short time.
+## 2022
+
+### Quarter 1
+
+- Brainstorm process
+
+---
+
+### Quarter 2
+
+- Initial project setup
+- First tests with RGW
+- [UI mockups][1]
+
+---
+
+### Quarter 3
+
+- File-based backend
+- Basic S3 operations support
+- User management
+- Helm charts
+- Testing
+- UI implementation
+- Documentation
+- Basic object management & versioning
+- Bucket management
+- Automation implementation
+
+---
+
+### Quarter 4
+
+- Object deletion/undeletion
+- Multipart uploads
+- Multipart copy
+- ACL support
+- UI: Basic object explorer
+- Rancher Partner chart
 
-Based on the user feedback we collect, we develop features that add up towards
-the goal of delivering the MVP. We're working on phases:
+## 2023
 
-## Phase 1
+### Plan of record
```

Comment:
I have no idea what `Plan of record` means in this context. It does seem to apply to the 2022 bullet points too though, so maybe just put it as a general paragraph at the top.